openaddr/__init__.py

```diff
@@ -148,8 +148,8 @@ def conform(source_config, destdir, extras):
 
     out_path = None
     if csv_path is not None and exists(csv_path):
```

Comment:
Change these variables from csv_path to output_path or something.

Reply:
Done, thanks!!